packages/playground/src/components/openapi-output.tsx

```diff
@@ -8,15 +9,19 @@ export interface OpenAPIOutputProps {
 export const OpenAPIOutput: FunctionComponent<OpenAPIOutputProps> = (props)=> {
   const[selected, setSelected] = useState<"raw"|"swagger-ui">("raw");
   const handleSelected = useCallback(
-    () => { return setSelected(selected === "raw"?("swagger-ui"):("raw")); },
+    () => { return setSelected(selected === "raw"?("swagger-ui"):("raw"));
+    },
     [selected],
   );
   return(
     <>
+    {PlaygroundManifest.enableSwaggerUI? (
     <select className="output-dropdown" onChange={handleSelected}>
       <option>OpenApi</option>
```

Comment:
should set the value on those options and mark which one is selected using the `selected` variable